src/audio/audio.cpp

```diff
@@ -464,6 +419,49 @@ void Audio::playAudioBuffer(ALuint alSource, const int16_t *data, int samples, u
         alSourcePlay(alSource);
 }
 
+void Audio::_cleanupInput()
+{
+    if (alInDev)
+    {
+#if (!FIX_SND_PCM_PREPARE_BUG)
+        qDebug() << "stopping capture";
+        alcCaptureStop(alInDev);
+#endif
+
+        if (alcCaptureCloseDevice(alInDev))
+        {
+            alInDev = nullptr;
```

Comment:
Because it is not calling `new` either -> have a look at `alcCaptureOpenDevice`. Looks like it "owns" the instance, but I am no OpenAL expert. :innocent: